sympy/solvers/inequalities.py

```diff
@@ -878,7 +878,7 @@ def classify(ie, s, i):
     conds.append(rv)
     return And(*conds)
 
-def _reduce_inequalities(inequalities, symbols):
+def _reduce_inequalities(inequalities, symbols, **flags):
```

Comment:
`**flags` is not used here.

Reply:
Yes, I'll just remove it in the next commit